### ISSUE TITLE
silero usage of `torch.stft`

### ISSUE THREAD

**Issue by vmoens** [pytorch/hub#283]
It seems that silero uses the `torch.stft` function in a way that will be deprecated in the next pytorch release.

Previously, the following usage was permitted
```python
torch.stft(tensor, num, num, num, num, bool, bool, *other)
```
but now a string representing the padding mode must be included

```python
torch.stft(tensor, num, num, num, num, bool, str, bool, *other)
```

[Link to the old function signature](https://github.com/pytorch/pytorch/blob/6f991fc5fcf993b26dffd8ebcbaba6f696d82eed/aten/src/ATen/autocast_mode.cpp#L544-L545)

[Link to the new function signature](https://github.com/pytorch/pytorch/blob/f2f924a90d8982a6369780662ea40a0c1e4483ea/aten/src/ATen/autocast_mode.cpp#L544)

[Link to the broken CI log](https://app.circleci.com/pipelines/github/pytorch/hub/1708/workflows/f5d8f3b8-4ad5-496e-8e6c-085f4ffbd619/jobs/2147)

[PR presumably responsible of the BC breaking change](https://github.com/pytorch/pytorch/pull/73434/files)

Perhaps @peterbell10 can comment on this?

@snakers4 do you think it's something that can be fixed? Because [the source code](https://github.com/snakers4/silero-models/issues/51#issuecomment-791216875) is not shared it's hard for us to debug this properly.

cc @NicolasHug

**Comment by snakers4** [comment]
Hi,

> [Link to the broken CI log](https://app.circleci.com/pipelines/github/pytorch/hub/1708/workflows/f5d8f3b8-4ad5-496e-8e6c-085f4ffbd619/jobs/2147)

Looks like this is based off a nightly PyTorch build.

Here is how we use `torch.stft` before converting to TorchScript

```
class STFT(nn.Module):
    def __init__(self):

....

    def forward(self, x):
        return torch.stft(x,
                          n_fft=self.n_fft,
                          hop_length=self.hop_length,
                          win_length=self.n_fft,
                          return_complex=True,
                          window=torch.hann_window(window_length=self.n_fft,
                                                   dtype=x.dtype, device=x.device)).abs()
```

>> but now a string representing the padding mode must be included
> pad_mode (string, optional) – controls the padding method used when center is True. Default: "reflect"

Is it already set in stone that this breaking change will be merged as it?
Usually PyTorch does not make such changes.

In any case the following files fail:

- file_path = 'python_code/snakers4_silero-vad_language.py'
- file_path = 'python_code/snakers4_silero-vad_number.py
- file_path = 'python_code/snakers4_silero-models_stt.py

In any case: `snakers4_silero-vad_language.py` and `python_code/snakers4_silero-vad_number.py` can be deprecated, because no one uses them and we decided to deprecate them.

And for `python_code/snakers4_silero-models_stt.py` I can just rebuild and reupload the latest models if the change gets added to a release.

**Comment by vmoens** [comment]
@snakers4 

> Is it already set in stone that this breaking change will be merged as is?
Usually PyTorch does not make such changes.

it seems this PR [will be reverted](https://github.com/pytorch/pytorch/pull/73434#issuecomment-1121516590) so let's wait for a couple of days before acting on anything.

Thanks for your responsiveness!

**Comment by NicolasHug** [comment]
Our CI seems to be green now: https://github.com/pytorch/hub/pull/225 (https://app.circleci.com/pipelines/github/pytorch/hub/1725/workflows/ec0914a3-1510-4dcd-b3b3-42f1eca174cd/jobs/2166)

Thanks a lot @vmoens for investigating, and @snakers4 for your timely replies!
